proto/felixbackend.pb.go

```diff
@@ -1,15797 +0,0 @@
-// Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
I don't think we should be deleting this file - is there something that go modules does that causes this?

Reply:
Hmm.. not sure how this happened, will revert it.